Add explanation on how to use xdebug (#28)
kellerkinderDE/devenv-shopware: README.md

- php81
- php82

## PHP debugging

> As mentioned in the features section XDebug is shipped with devenv, but you need to activate it to actully debug something. For the Frontend you can use the [XDebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively you can simply add a query parameter to the URL you want to debug like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`

Reviewer: ```suggestion
As mentioned in the features section, XDebug is shipped with devenv and you need to activate it to actually debug something. For the Frontend you can use the [XDebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively, you can simply add a query parameter to the URL you want to debug like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`
```

Author: Thank you! Applied.

> This also works great to debug API request in Insomnia or something similar.

Reviewer: ```suggestion
This also works great when having to debug API reques, e.g in Insomnia.
```

Reviewer 2: I like this more then my suggestion to line 168. 👍

Author: Thank you! Applied.

> As mentioned in the features section XDebug is shipped with devenv, but you need to activate it to actully debug something. For the Frontend you can use the [XDebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively you can simply add a query parameter to the URL you want to debug like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`
> This also works great to debug API request in Insomnia or something similar.

Reviewer 2: [Xdebug](https://xdebug.org/) itself uses "Xdebug".
```suggestion
As mentioned in the features section Xdebug is shipped with devenv, but you need to activate it to actually debug something. For the Frontend you can use the [Xdebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively you can simply add a query parameter to the URL you want to debug. Like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`
This also works great to debug API requests in Insomnia or something similar.
```

> For the CLI you can use an integrated debug helper by simply adding `debug` before your CLI commands, e.g. `debug bin/console`.

Reviewer: ```suggestion
In the CLI you can use an integrated debug helper by simply adding `debug` before your CLI commands, e.g. `debug bin/console`.
```

> For the CLI you can use an integrated debug helper by simply adding `debug` before your CLI commands, e.g. `debug bin/console`.

Reviewer 2: ```suggestion
For the CLI you can use the integrated debug helper `debug` by simply adding it before your CLI commands, e.g. `debug bin/console`.
```

Author: Thank you! Applied.
